managed/ui/src/components/config/Storage/BackupList.js

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) YugaByte, Inc.
+//
+// Author: Nishant Sharma(nishant.sharma@hashedin.com)
+//
+// This file will hold a common list view for the different kind
+// of storage configuration.
+
+import React, { useState } from "react";
+import { Button, DropdownButton, MenuItem } from "react-bootstrap";
+import { BootstrapTable, TableHeaderColumn } from "react-bootstrap-table";
+import YBInfoTip from "../../common/descriptors/YBInfoTip";
+import { FlexContainer, FlexShrink } from "../../common/flexbox/YBFlexBox";
+import { YBConfirmModal } from "../../modals";
+import { YBPanelItem } from "../../panels";
+
+/**
+ * This method is used to show the config name on the
+ * list view.
+ * 
+ * @param {string} cell Not in-use.
+ * @param {object} row Respective row details.
+ * @returns Config name
+ */
+const getBackupConfigName = (cell, row) => {
+  return row.configName;
+};
+
+/**
+ * This method is used to return the current in-use status
+ * of the backup storage config.
+ * 
+ * @param {string} cell Not in-use.
+ * @param {object} row Respective row details.
+ * @returns "Used" || "Not Used".
+ */
+const getBackupStatus = (cell, row) => {
+  return row.inUse ? "Used" : "Not Used";
+};
+
+/**
+ * This method is used to return the backup location of
+ * respective backup config.
+ * 
+ * @param {string} cell Not in-use.
+ * @param {object} row Respective row details.
+ * @returns Backup storage location.
+ */
+const getBackupLocation = (cell, row) => {
+  return row.data.BACKUP_LOCATION;
+};
+
+/**
+ * This method will handle all the required actions for
+ * the particular row.
+ * 
+ * @param {string} cell Not in-use.
+ * @param {object} row Respective row details.
+ * @param {object} operations Actions.
+ * @returns Actions.
+ */
+const foramtConfigActions = (cell, row, operations) => {
+  const {
+    configUUID,
+    inUse,
+    name
+  } = row;
+  const {
+    configData,
+    onEdit,
+    deleteStorageConfig,
+    setConfigData,
+    showDeleteStorageConfig,
+    visibleModal,
+    hideDeleteStorageConfig
+  } = operations;
+
+  return (
+    <DropdownButton
+      className="backup-config-actions btn btn-default"
+      title="Actions"
+      id="bg-nested-dropdown"
+      pullRight
+    >
+      <MenuItem onClick={() => onEdit(row)}>
+        <i className="fa fa-pencil"></i> Edit Configuration
+      </MenuItem>
+      <MenuItem
+        onClick={() => {
+          setConfigData(configUUID);
+          showDeleteStorageConfig(name);
+        }}
+        disabled={inUse}>
+        {!inUse &&
```

Comment:
smth wrong with indents and `>` placement here, pls fix

Reply:
Done